mne/viz/ica.py

```diff
@@ -329,17 +356,50 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
 
     # calculations
     # ------------
+
     if isinstance(inst, BaseRaw):
+        # if reject is set, reject
```

Comment:
I would remove the comment. But if you think its needed, I would change it for this:

`# when auto, delegate reject to the ica`

and then I would add an empty line to facilitate the reading. So that is clear that I do 2 things:
1 - decide what reject
2 - act on the reject

otherwise, It gets confused with a `switch case` that.

So I would write:
```py
# when auto, delegate reject to the ica
if reject == 'auto':
    reject = getattr(ica, 'reject_', None)
else:
    pass

if reject is None:
    drop_inds = None
    inst_rejected = inst
else:
    data = inst.get_data()
    data, drop_inds = _reject_data_segments(data, ica.reject_,
                                            flat=None, decim=None,
                                            info=inst.info,
                                            tstep=2.0)
    inst_rejected = RawArray(data, inst.info)
```

You can drop the `else pass`.